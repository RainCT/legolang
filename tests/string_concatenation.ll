print "Hi " + "my" + " " + "friend!"
print "This is a number: " + 5 + "."
print 42 + " is another number."

// Same with variables
print ""

string name = "John"
int number = 110

print "Hi, " + name + "!"
print number + " is a number, just like " + (number + 1) + "."
