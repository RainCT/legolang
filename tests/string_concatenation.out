Hi my friend!
This is a number: 5.
42 is another number.

Hi, John!
110 is a number, just like 111.
